particles/Renderer.cpp

```diff
@@ -168,10 +156,6 @@ void Renderer::init(score::gfx::RenderList& renderer)
     {
         vec4 spd;
     };
-    struct Ctrl
```

Comment:
I simplified this a bit, this is not necessary when we have uniform buffers, the ubo is already kind of a struct. (also this did not work on my graphics card)